sounddevice.py

```diff
@@ -2289,13 +2289,40 @@ def reset(self):
         vars(self).clear()
         self.__init__()
 
+
 if not hasattr(_ffi, 'I_AM_FAKE'):
     # This object shadows the 'default' class, except when building the docs.
     default = default()
 
 
 class PortAudioError(Exception):
-    """This exception will be raised on PortAudio errors."""
+    """This exception will be raised on PortAudio errors.
+
+    Attributes
+    ----------
+    args
+        A variable length tuple containing the following elements when
+        available:
+
+        1) The user error message
```

Comment:
heh, actually it's kind of vague. I meant "the error message that get's presented to the user" but that's not really correct either. I didn't want to say "The PortAudio error message" since that's also not totally correct, but maybe it's close enough. Would that be alright?